docs/docs/queries/dql-js-inline.md

```diff
@@ -57,10 +57,16 @@ Today is November 07, 2022 - 2 months, 5 days until exams!
 `= [[secondPage]].someMetadataField`
 ~~~
 
-You can use everything available as [expressions](../../reference/expressions) and [literals](../../reference/literals) in a Inline DQL, including [functions](../../reference/functions). Query Types and Data Commands on the other hand are **not available in Inlines.**
+You can use everything available as [expressions](../../reference/expressions) and [literals](../../reference/literals) in an Inline DQL Query, including [functions](../../reference/functions). Query Types and Data Commands on the other hand are **not available in Inlines.**
```

Comment:
Comma before and after "on the other hand". I wonder if we should have a grammar linter or something haha.